template/web/src/providers/OnchainProviders.tsx

```diff
@@ -41,20 +42,20 @@ const connectors = connectorsForWallets(
   },
 );
 
-const wagmiConfig = createConfig({
-  ssr: true,
-  chains: [baseSepolia, base],
-  transports: {
-    [baseSepolia.id]: http(),
-    [base.id]: http(),
-  },
-  connectors,
-});
-
 /**
  * TODO Docs ~~~
  */
 function OnchainProviders({ children }: Props) {
+  const wagmiConfig = createConfig({
```

Comment:
Not sure this is the right move.

What happens if the Provider components re-render, I assume the `createConfig` function will be called again.

Reply:
I think is best run this outiside the React life-cycle. So that also it starts the execution right away.

Reply:
resolved